src/Core/AndroidNative/maui/build.gradle

```diff
@@ -31,5 +31,6 @@ dependencies {
     annotationProcessor 'com.github.bumptech.glide:compiler:4.15.1'
 
     implementation 'androidx.appcompat:appcompat:1.3.1'
+    implementation 'androidx.window:window:1.1.0'
```

Comment:
It is OK to add this dependency in Java, because we already have it in C#:

https://github.com/dotnet/maui/blob/8203e4d25e108550ecabdc7cf7160d0257582fa8/eng/AndroidX.targets#L60-L63

We were using the same APIs in C# previously.